frame/staking/src/tests.rs

```diff
@@ -1905,13 +1906,13 @@ fn bond_with_duplicate_vote_should_be_ignored_by_election_provider_elected() {
 
 			// winners should be 21 and 11.
 			let supports = <Test as Config>::ElectionProvider::elect().unwrap().0;
-			assert_eq!(
-				supports,
-				vec![
+			assert!(supports_eq_unordered(
```

Comment:
`assert_eq_uvec!` is what you need here.

Reply:
Unfortunately, it's not quite that straightforward. We don't care about the ordering of the `supports` in the outer vector, but we _also_ don't care about the ordering of `support.voters`.